dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -198,3 +205,9 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
+
+http.compression.types=List of content types for response compression
+http.compression.types.desc=Specifies a list of content types for which the HTTP Channel will perform compression of the response if able.
```

Comment:
**change**
Specifies a list of content types for which the HTTP Channel will perform compression of the response if able.
**to**
Specifies a list of content types for which the HTTP Channel  compresses the response if it is able to do so.

Reply:
Added some more description to include that the (+) and (-) characters can prefix an element of this list to either add it or remove from the list. Not using the (+) character overwrites the default values.